benchs/base.py

```diff
@@ -126,8 +126,8 @@ def get_sector_size(self, dev):
 
         return sectorsize
 
-    def get_nvme_drive_capacity_gb(self, path):
-        if is_dev_zoned(path):
+    def get_nvme_drive_capacity_gb(self, dev, path):
+        if is_dev_zoned(dev):
             filename = path + "/blkzone-capacity.txt"
```

Comment:
I am suggesting that we do not introduce the `dev` parameter within this function (and therefore each `report` function).
The reason is that the `get_nvme_drive_capacity_gb` is being called in the report function of `fio_zone_writes.py` and `rocksdb.py`.
A report can separately be generated (after `run` finished) without the need to specify the device (see `README.md`).

The output directory should collect all valuable information about the run, including if the device was zoned.
My suggestion is to check if `blkzone-capacity.txt` exists because this file is only generated when the benchmark was run on a zoned block device:
```suggestion
    def get_nvme_drive_capacity_gb(self, dev, path):
        filename = path + "/blkzone-capacity.txt"
        zoned_dev = os.path.exists(filename)
        if zoned_dev :
```